vformer/attention/cross.py

```diff
@@ -73,3 +72,57 @@ def forward(self, cls, patches):
         ycls = self.gl(ycls)
 
         return ycls
+
+
+@ATTENTION_REGISTRY.register()
+class CrossAttention(nn.Module):
+    """
+    Cross-Attention
+
+    Parameters
+    ----------
+    query_dim: int
+        Dimension of query array
+    context_dim: int
+        Dimension of context array
+    num_heads: int
+        Number of cross-attention heads
+    head_dim: int
+        Dimension of each head
+
+    """
+
+    def __init__(self, query_dim, context_dim, num_heads=8, head_dim=64):
+        super().__init__()
+
+        inner_dim = num_heads * head_dim
+        self.num_heads = num_heads
+        self.scale = head_dim ** -0.5
+        self.to_q = nn.Linear(query_dim, inner_dim)
+        self.to_k = nn.Linear(context_dim, inner_dim)
+        self.to_v = nn.Linear(context_dim, inner_dim)
+        self.to_out = _Projection(inner_dim, query_dim)
```

Comment:
Minor nit:
I dont think we need to have another class for this, we can do the same thing with if else statement-
`self.to_out = nn.Linear(inner_dim, query_dim) if inner_dim == query_dim else nn.Identity()`

Reply:
fixed with [27cb828](https://github.com/SforAiDl/vformer/pull/79/commits/27cb82886fbb715b0aa7ef7e6ce5eaa9577593d6)